### ISSUE TITLE
[Assistive tech and device support] Page functionality isn't keyboard accessible. (11.05.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#34161]
#### VFS team name
Debt Resolution

#### VFS product name
Medical Copays (MCP)

#### Point of Contact/Reviewers
Brian DeConinck (@briandeconinck) - Accessibility

---

### Platform Issue
Page functionality isn't keyboard accessible.

### Issue Details
This issue occurs with headings in three places that I've noticed:
- When not logged in, the heading "Please sign in to review your VA copay balances" in the `<va-alert>` component
- When logged in, "Review your VA copay balances" in the `<va-alert>` component
- When on the /balance-details/ page, the H2 heading in the `<va-alert>` component

When these headings are announced by NVDA, they're announced as "clickable heading". However, the headings cannot receive keyboard focus or be interacted with using [Enter] or [Spacebar]. If there's an action associated with clicking on those headings, keyboard accessibility should be supported.

If there's not an action associated with clicking on those headings, then this isn't really a keyboard accessibility issue --- and as far as I can tell there's no click action that's actually happening. My best guess is there's an extra JS event listener that's listening for clicks but doesn't actually do anything with them, and that's throwing NVDA off. Since this seems to only be happening with headings in the `<va-alert>` component it might also end up being a design system team issue.

### Link, screenshot or steps to recreate
1. Navigate to https://staging.va.gov/health-care/pay-copay-bill/
2. Activate NVDA and announce main page content.
3. When NVDA reaches the second alert on the page, it will be announced as `Clickable Heading Level 2, Please sign in to review your VA copay balances`.

### VA.gov Experience Standard
[Category Number 11, Issue Number 05](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.1.1_A

---

### Platform Recommendation
If there's an intended clickable action on those headings, add tabindex=0 to support keyboard focus and ensure that [Enter] and [Spacebar] are supported. 

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by shiragoodman** [comment]
@briandeconinck please take a look to expand the issue description. Thanks!

**Comment by craigwheeler** [comment]
this is great thanks for the clarification @briandeconinck 👍

**Comment by craigwheeler** [comment]
@briandeconinck after digging into this I was able to verify that these issues are with the [va-alert](https://design.va.gov/components/alertboxes) design system component thats being used and not directly related to MCP can you advise what we should do next?

**Comment by briandeconinck** [comment]
@craigwheeler I believe Shira will be reassigning this issue to the appropriate team. Thank you!

**Comment by shiragoodman** [comment]
@caw310 per the comments above, we have reassigned this ticket to the Design System team. Please do not close this ticket out until the issue has been resolved as we use this ticket to populate our Collaboration Cycle dashboard. Let me know if you have any questions.

**Comment by andresriveratoro** [comment]
It looks like the problem is that the heading is inside a div that is clickable.
